redirects.json

```diff
@@ -224,8 +224,8 @@
         ]
     },
     {
-        "to": "/get-started/tooling/libraries/walletconnect",
-        "from": "/developers/tooling/libraries/wallet-connect"
+        "to": "/get-started/tooling/libraries/reown",
+        "from": "/developers/tooling/libraries/reown"
```

Comment:
The redirects need to account for the old path, not the new one with `reown`. I'll see if I can push up a change to correct this.